explainer.md

```diff
@@ -622,19 +614,24 @@ partial interface Navigator {
   readonly attribute VR vr;
 };
 
-[SecureContext, Exposed=Window] interface VR : EventTarget {
-  attribute EventHandler ondeviceconnect;
-  attribute EventHandler ondevicedisconnect;
+dictionary VRDeviceFilter {
+  boolean exclusive = false;
+};
+
+dictionary VRDeviceRequestOptions {
+  required sequence<VRDeviceFilter> filters;
+};
 
-  Promise<sequence<VRDevice>> getDevices();
+[SecureContext, Exposed=Window] interface VR : EventTarget {
```

Comment:
can we have this (and others) exposed on `(Window,Worker)`? or do you feel that warrants a separate issue and discussion?

Reply:
Definitely want to get there, but I do feel it's worth having it's own issue. I don't _expect_ it to be controversial, but any conversation that does crop up around it deserves its own space rather than being intermixed with this change.